changelogs/fragments/3041-gitlab_x_members_fix_and_enhancement.yml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
+- "fix of gitlab_group_members which had an issue with upstream api pagination leading to failing when more then 20 users in gitlab group."
+- "fix of gitlab_project_members which had an issue with upstream api pagination leading to failing when more then 20 users in gitlab project."
```

Comment:
Some first quick comments:
1. Please stick to the **exact** format for changelog fragments, see https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to for details.
2. Bugfixes should be in a separate PR than features. Bugfixes will be backported to stable-2 and stable-3, while features are only backported to stable-3.

Reply:
Created a bugfix only PR (https://github.com/ansible-collections/community.general/pull/3054). Changed this PR to reflect the new feature only.